gadget/gadget.go

```diff
@@ -447,40 +447,34 @@ func validateVolume(name string, vol *Volume, constraints *ModelConstraints) err
 	return validateCrossVolumeStructure(structures, knownStructures)
 }
 
-func ensureVolumeConsistency(state *validationState, constraints *ModelConstraints) error {
-	// system-seed also requires system-data
-	if state.SystemSeed != nil && state.SystemData == nil {
+func ensureVolumeConsistencyNoConstraints(state *validationState, constraints *ModelConstraints) error {
```

Comment:
this doesn't need to take constraints, no?

Reply:
Pushed a fix for this.